cypress/support/config/services.js

```diff
@@ -90,7 +90,7 @@ module.exports = {
           Cypress.env('APP_ENV') === 'live'
             ? undefined
             : '/amharic/bbc_amharic_radio/liveradio',
-        smoke: false,
+        smoke: true,
```

Comment:
We mustn't do this as it runs all tests on these pages. There is an easier way to do this.

You write the test hardcoded in the testsThatAlwaysRunForCanonicalOnly to only run on one of each service with a different logic.

An unrelated concern is that this e2e overlaps I think entirely with one written in https://github.com/bbc/simorgh/pull/3339 and I believe that PR covers all page types, whereas this is page specific, so the other is preferable.

Reply:
Ah so that is why only one service's live radio smoke tests were enabled? Makes sense to me now.